<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>794</width>
    <height>664</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>650</width>
    <height>650</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout" stretch="3,1,7">
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QWidget" name="topHLayoutMainPage" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>13</width>
             <height>31</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="rmAllBtn_">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="text">
            <string>Clear All</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditSearch_">
           <property name="minimumSize">
            <size>
             <width>250</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cmbSearchItems_">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="searchBtn_">
           <property name="minimumSize">
            <size>
             <width>80</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="text">
            <string>Search</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="plusBtn_">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="cursor">
            <cursorShape>PointingHandCursor</cursorShape>
           </property>
           <property name="autoFillBackground">
            <bool>false</bool>
           </property>
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Save Search</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>37</width>
             <height>17</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="toolLayoutMainPage" native="true">
      <property name="styleSheet">
       <string notr="true"> QPushButton{
	border-radius: 5px;
   border: 1px solid  rgb(0, 11, 85);
	background-color: rgb(93, 109, 255);
    border: 1px solid darkerblue;
	padding:  0 8px;
 }</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Filter:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lneFilter">
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QSplitter" name="splitterMainViews">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="AXListView" name="lblListView_">
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>Times New Roman</family>
         <pointsize>12</pointsize>
         <italic>false</italic>
         <bold>false</bold>
        </font>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="tabKeyNavigation">
        <bool>true</bool>
       </property>
       <property name="dragEnabled">
        <bool>false</bool>
       </property>
       <property name="dragDropOverwriteMode">
        <bool>false</bool>
       </property>
       <property name="dragDropMode">
        <enum>QAbstractItemView::DragOnly</enum>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="movement">
        <enum>QListView::Free</enum>
       </property>
       <property name="resizeMode">
        <enum>QListView::Fixed</enum>
       </property>
       <property name="layoutMode">
        <enum>QListView::SinglePass</enum>
       </property>
       <property name="gridSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="uniformItemSizes">
        <bool>true</bool>
       </property>
       <property name="selectionRectVisible">
        <bool>true</bool>
       </property>
       <property name="itemAlignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="AXTableView" name="searchResultView_">
       <property name="mouseTracking">
        <bool>true</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AXTableView</class>
   <extends>QTableView</extends>
   <header>Headers/Views/axtableview.h</header>
  </customwidget>
  <customwidget>
   <class>AXListView</class>
   <extends>QListView</extends>
   <header location="global">Headers/Views/axlistview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
